vspace/pom.xml

```diff
@@ -12,7 +12,7 @@
         <org.slf4j-version>1.7.5</org.slf4j-version>
         <spring.version>5.0.19.RELEASE</spring.version>
         <spring-data.version>Kay-SR8</spring-data.version>
-        <spring-security-version>5.3.10.RELEASE</spring-security-version>
+        <spring-security-version>5.3.3.RELEASE</spring-security-version>
```

Comment:
is there a reason you downgraded this?

Reply:
I think this has happened accidentally and committed. I have made changes now to correct this to latest.